app/helpers/react_on_rails_helper.rb

```diff
@@ -86,14 +86,17 @@ def debug_js(react_component, data_variable, dom_id, trace)
     end
   end
 
+  def render_js_react_element(react_component, props)
+    ReactOnRails::ReactRenderer.new.render_js_react_element(react_component, props).html_safe
```

Comment:
1. Why html safe? Shouldn't be required. The output of the entire helper needs to be marked as html_safe.
2. Why not make the ReactRenderer.render_js_react_element static?